### PR TITLE
Updating to account for new contributors guide.

### PR DESCRIPTION
**Pull request by moxiegirl**
Primary links will go to the contributor's guide.
Funnel newbies to detail guide but leave behind some quick tips for impatient experienced folks.
Reduced some redundancy in language.
Sentence case all the headings. 

@docker/maintainers and so forth...please to review. :-D

Signed-off-by: Mary Anthony mary.anthony@docker.com

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,70 +1,55 @@
 # Contributing to Docker
 
-Want to hack on Docker? Awesome! Here are instructions to get you
-started. They are probably not perfect; please let us know if anything
-feels wrong or incomplete.
+Want to hack on Docker? Awesome!  We have a contributor's guide that explains [setting up a Docker development environment and the contributing process](https://docs.docker.com/project/who-written-for/).
```
Comment (fredlf) [inline CONTRIBUTING.md]: "contribution process"

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,70 +1,55 @@
...
-* A description of the problem this design proposal solves
-* A pull request, not an issue, that modifies the documentation describing
-  the feature you are proposing, adding new documentation if necessary.
-  * Please prefix your issue with `Proposal:` in the title
-* Please review [the existing Proposals](https://github.com/docker/docker/pulls?q=is%3Aopen+is%3Apr+label%3AProposal)
-  before reporting a new one. You can always pair with someone if you both
-  have the same idea.
+Security reports are greatly appreciated and we will publicly thank you for it.
+We also like to send gifts &mdash; if you're into Docker schwag make sure to let
+us know :) We currently do not offer a paid security bounty program, but are not
```
Comment (fredlf) [inline CONTRIBUTING.md]: Not sure about emoticons in docs, but lean towards not having them. Seems a slippery slope. 

Dash and semi-colon are redundant.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,70 +1,55 @@
...
+## Reporting other issues
 
 A great way to contribute to the project is to send a detailed report when you
 encounter an issue. We always appreciate a well-written, thorough bug report,
 and will thank you for it!
 
-When reporting [issues](https://github.com/docker/docker/issues) on
-GitHub please include your host OS (Ubuntu 12.04, Fedora 19, etc).
-Please include:
+Check that [a our issue database](https://github.com/docker/docker/issues)
```
Comment (fredlf) [inline CONTRIBUTING.md]: Superfluous "a"

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -1,70 +1,55 @@
...
+doesn't already include that problem or suggestion before submitting an issue.
+If you find a match, add a quick "+1" or "I have this problem too." Doing this
+helps prioritize the most common problems and requests.
+
+When reporting issues please include your host OS (Ubuntu 12.04, Fedora 19,
```
Comment (fredlf) [inline CONTRIBUTING.md]: Comma needed after "issues".

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -103,121 +88,156 @@ Additional info:
...
+
+###Pull requests are always welcome
+
+Not sure if that typo is worth a pull request? Found a bug and know how to fix
+it? Do it! We will appreciate it. Any significant improvement should be
+documented as [a GitHub issue](https://github.com/docker/docker/issues) before
+anybody starts working on it.
+
+We are always thrilled to receive pull requests. We do our best to process them
+as quickly. If your pull request is not accepted on the first try,
```
Comment (fredlf) [inline CONTRIBUTING.md]: Missing words. "...as quickly as possible." (I'm assuming.)

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -103,121 +88,156 @@ Additional info:
...
+  <col width="45%">
+  <col width="65%">
+  <tr>
+    <td>Internet&nbsp;Relay&nbsp;Chat&nbsp;(IRC)</th>
+    <td>
+      <p>
+        IRC a direct line to our most knowledgeable Docker users; we have
+        both the  <code>#docker</code> and <code>#docker-dev</code> group on 
+        <strong>irc.freenode.net</strong>. IRC was first created in 1988. 
+        So, it is a rich chat protocol but it can overwhelm new users. You can search
```
Comment (fredlf) [inline CONTRIBUTING.md]: I think there's a thought missing here. What's the relationship between IRC's creation date and the fact that it is a rich protocol?

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -103,121 +88,156 @@ Additional info:
...
+      is for people using Docker containers. 
+      The <a href="https://groups.google.com/forum/#!forum/docker-dev" target="_blank">docker-dev</a> 
+      group is for contributors and other people contributing to the Docker 
+      project.
+    </td>
+  </tr>
+  <tr>
+    <td>Twitter</td>
+    <td>
+      You can follow <a href="https://twitter.com/docker/" target="_blank">Docker's twitter</a>
```
Comment (fredlf) [inline CONTRIBUTING.md]: Twitter feed is more correct, I think (but I don't Twitter, so...)

File: CONTRIBUTING.md [hunk 8]
```diff
@@ -103,121 +88,156 @@ Additional info:
...
+      to get updates on our products. You can also tweet us questions or just 
+      share blogs or stories.
+    </td>
+  </tr>
+  <tr>
+    <td>Stack Overflow</td>
+    <td>
+      Stack Overflow has over 7000K Docker questions listed. We regularly
```
Comment (fredlf) [inline CONTRIBUTING.md]: 7000k? Really? Whoa. :-)

File: CONTRIBUTING.md [hunk 9]
```diff
@@ -103,121 +88,156 @@ Additional info:
...
-Update the documentation when creating or modifying features. Test
-your documentation changes for clarity, concision, and correctness, as
-well as a clean documentation build. See `docs/README.md` for more
-information on building the docs and how they get released.
+Update the documentation when creating or modifying features. Test your
+documentation changes for clarity, concision, and correctness, as well as a
+clean documentation build. See `docs/README.md` for more information on
+[building the
+docs](https://docs.docker.com/project/test-and-docs/#build-and-test-the-
+documentation) and how they get released.
```
Comment (fredlf) [inline CONTRIBUTING.md]: Might want to add a link to Style Guide too. I'm really hoping we can get people familiar with it before they start documenting.

File: CONTRIBUTING.md [hunk 10]
```diff
@@ -103,121 +88,156 @@ Additional info:
...
 
 Write clean code. Universally formatted code promotes ease of writing, reading,
 and maintenance. Always run `gofmt -s -w file.go` on each changed file before
 committing your changes. Most editors have plug-ins that do this automatically.
 
-Pull requests descriptions should be as clear as possible and include a
-reference to all the issues that they address.
+Pull requests descriptions should be as clear as possible and include a reference
```
Comment (fredlf) [inline CONTRIBUTING.md]: Typo or missing apostrophe: "requests"

File: CONTRIBUTING.md [hunk 11]
```diff
@@ -103,121 +88,156 @@ Additional info:
...
+suggested modifications and push additional commits to your feature branch. Post
+a comment after pushing. New commits show up in the pull request automatically,
+but the reviewers are notified only when you comment.
 
-Pull requests must be cleanly rebased ontop of master without multiple branches
+Pull requests must be cleanly rebased on top of master without multiple branches
 mixed into the PR.
 
 **Git tip**: If your PR no longer merges cleanly, use `rebase master` in your
 feature branch to update your pull request rather than `merge master`.
```
Comment (fredlf) [inline CONTRIBUTING.md]: We mIght want to explain why this is so, since many people are scared of rebase.

File: CONTRIBUTING.md [hunk 12]
```diff
@@ -103,121 +88,156 @@ Additional info:
...
 
-Before the pull request is merged, make sure that you squash your commits into
-logical units of work using `git rebase -i` and `git push -f`. After every
-commit the test suite should be passing. Include documentation changes in the
-same commit so that a revert would remove all traces of the feature or fix.
+Before you make a pull request, squash your commits into logical units of work
+using `git rebase -i` and `git push -f`. After every commit, make sure the test
+suite passes. Include documentation changes in the same pull request so that a
```
Comment (fredlf) [inline CONTRIBUTING.md]: Might want to briefly tell people how tests are run and passed automatically, so they don't think they are missing info.

File: CONTRIBUTING.md [hunk 13]
```diff
@@ -103,121 +88,156 @@ Additional info:
...
+For more details, see the [MAINTAINERS](MAINTAINERS) page.
 
 ### Sign your work
 
 The sign-off is a simple line at the end of the explanation for the
 patch, which certifies that you wrote it or otherwise have the right to
-pass it on as an open-source patch.  The rules are pretty simple: if you
+pass it on as an open-source patch. The rules are pretty simple: if you
 can certify the below (from
 [developercertificate.org](http://developercertificate.org/)):
```
Comment (fredlf) [inline CONTRIBUTING.md]: Not a complete sentence, missing independent clause.

File: CONTRIBUTING.md [hunk 14]
```diff
@@ -280,45 +300,43 @@ format right away, but please do adjust your processes for future contributions.
...
-Anybody can ask for a topic to be discussed prior to the meeting.
+There are two monthly meetings taking place on #docker-dev IRC to accomodate all
+timezones. Anybody can propose a topic for discussion prior to the meeting.
 
 If you feel the conversation is going off-topic, feel free to point it out.
 
-For the exact dates and times, have a look at [the irc-minutes repo](https://github.com/docker/irc-minutes).
-They also contain all the notes from previous meetings.
+For the exact dates and times, have a look at [the irc-minutes
+repo](https://github.com/docker/irc-minutes). They also contain all the notes
```
Comment (fredlf) [inline CONTRIBUTING.md]: "They" should be "it", unless I'm misunderstanding something.

**Comment by fredlf** [comment]
A few suggestions and a bunch of copy-edits. LGTM once copy-edits are fixed.

File: CONTRIBUTING.md (hunk 2 above)
Comment (moxiegirl) [inline CONTRIBUTING.md]: The emoticon was in the original. I was trying to preserve the original tone. I'll take it out.   The semi-colon is part of the HTML code. Doesn't exist in the actual text.

File: CONTRIBUTING.md (hunk 11 above)
Comment (moxiegirl) [inline CONTRIBUTING.md]: Merge versus rebase is not a short blurb --- especially if the reader is afraid of rebase.

File: CONTRIBUTING.md [hunk 15]
```diff
@@ -280,45 +307,43 @@ format right away, but please do adjust your processes for future contributions.
...
-need your help to keep it that way. To help with this we've come up with some
-general guidelines for the community as a whole:
+We want to keep the Docker community awesome, growing and collaborative. We need
+your help to keep it that way. To help with this we've come up with some general
+guidelines for the community as a whole:
 
 * Be nice: Be courteous, respectful and polite to fellow community members: no
-  regional, racial, gender, or other abuse will be tolerated. We like nice people
-  way better than mean ones!
+regional, racial, gender, or other abuse will be tolerated. We like nice people
```
Comment (thaJeztah) [inline CONTRIBUTING.md]: Nit; would it be possible to keep the hanging indent in the bullet list? It makes the document easier to read when viewing it as plain text, e.g. in an editor.

File: CONTRIBUTING.md [hunk 16]
```diff
@@ -1,70 +1,61 @@
...
 
-* A description of the problem this design proposal solves
-* A pull request, not an issue, that modifies the documentation describing
-  the feature you are proposing, adding new documentation if necessary.
-  * Please prefix your issue with `Proposal:` in the title
-* Please review [the existing Proposals](https://github.com/docker/docker/pulls?q=is%3Aopen+is%3Apr+label%3AProposal)
-  before reporting a new one. You can always pair with someone if you both
-  have the same idea.
+Security reports are greatly appreciated and we will publicly thank you for it.
+We also like to send gifts&mdash;if you're into Docker schwag make sure to let
```
Comment (thaJeztah) [inline CONTRIBUTING.md]: Minor nit; spaces surrounding the em dash? (Wasn't sure but found it here [AP style guide](http://www.apvschicago.com/2011/05/em-dashes-and-ellipses-closed-or-spaced.html))

**Comment by moxiegirl** [comment]
Nits picked up and kicked out.

**Comment by moxiegirl** [comment]
@thaJeztah I'll pick that up next time I'm in the file. I kinda want to get this out. Takes a bit to build.

**Comment by thaJeztah** [comment]
No problem at all, I realise there's pressure for the bday party. Just noted the issues while reading

**Comment by crosbymichael** [comment]
LGTM

File: CONTRIBUTING.md (hunk 11 above)
Comment (SvenDowideit) [inline CONTRIBUTING.md]: mmm, rebasing to master is pretty dangerous - compared to rebasing to `upstream/master`

File: CONTRIBUTING.md [hunk 17]
```diff
@@ -280,45 +307,45 @@ format right away, but please do adjust your processes for future contributions.
...
+will have time to make yourself available. You don't have to be a
 maintainer to make a difference on the project!
 
-### IRC Meetings
+### IRC meetings
 
-There are two monthly meetings taking place on #docker-dev IRC to accomodate all timezones.
-Anybody can ask for a topic to be discussed prior to the meeting.
+There are two monthly meetings taking place on #docker-dev IRC to accomodate all
+timezones. Anybody can propose a topic for discussion prior to the meeting.
```
Comment (thaJeztah) [inline CONTRIBUTING.md]: What is the procedure to propose a topic for discussion?

(can be added in a follow-up PR)

**Pull request merged**